driver/network_driver.go

```
@@ -205,9 +214,37 @@ func (d NetworkDriver) CreateNetwork(request *network.CreateNetworkRequest) erro
 			log.Errorln(err)
 			return err
 		}
+		ps = append(ps, ipData.Pool)
 	}
 
 	logutils.JSONMessage("CreateNetwork response", map[string]string{})
+	return d.populatePoolLabel(ps, request.NetworkID)
+}
+
+func (d NetworkDriver) populatePoolLabel(pools []string, networkID string) error {
+	ctx := context.Background()
+	poolClient := d.client.IPPools()
+	ipPools, err := poolClient.List(ctx, options.ListOptions{})
+	if err != nil {
+		log.Errorln(err)
+		return err
+	}
+	for _, ipPool := range ipPools.Items {
+		for _, cidr := range pools {
+			if ipPool.Spec.CIDR == cidr {
+				ann := ipPool.GetAnnotations()
+				if ann == nil {
+					ann = map[string]string{}
+				}
+				ann[DOCKER_LABEL_PREFIX+"network.ID"] = networkID
+				ipPool.SetAnnotations(ann)
+				_, err = poolClient.Update(ctx, &ipPool, options.SetOptions{})
+				if err != nil {
+					log.Errorln(err)
```

Shouldn't we return an error here?